### PR TITLE
Code reviews between coding partners

### PR DESCRIPTION
**Pull request by MohamedHNoor**
# Code Review between @MohamedHNoor  and @yash244466666 

- [x] StudentA - @MohamedHNoor 
- [x] StudentB - @yash244466666 

- We have implemented all the requirements for the project 
-  Now we want to do code review on our work

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,9 +1,382 @@
...
+
+/* footer */
+
+.footer {
+  padding: 5px 20px;
+  font-size: 18px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (MohamedHNoor) [inline src/style.css:219]: Please  make the footer alignment to the left as it's shown in the WireFrames

**Review by MohamedHNoor — COMMENTED** [review]
# Highlights ♻️
- Well done 👍  for completing your part of the project @yash244466666

     - [x] Used ES6 syntax.
     - [x] Used API documentation
     - [x] Send and receive data from an API.
     - [x] Make JavaScript code asynchronous.

# Changes Required:
  - Please see the comments below and apply the changes. Thank you

**Comment by yash244466666** [comment]
@MohamedHNoor Done fixing the alignment issue with the footer  text

**Review by yash244466666 — APPROVED** [review]
# Highlights

- [x] Use ES6 syntax.
- [x]  Use Webpack.
- [x]  Follow Gitflow
- [x] Use ES6 modules.
- [x] Follow Gitflow.
- [x] Solve simple git conflicts.
- [x] Make JavaScript code asynchronous.
- [x] Send and receive data from an API.


I have to say that everything is perfect. You did a great job. Your code is approved ✔✔✔✔

**Review by MohamedHNoor — COMMENTED** [review]
# Highlights ♻️ 

- [x] Well done @yash244466666  your comment count test is working.
- [x] I approved your code. Thank you for the hard work

**Review by yash244466666 — APPROVED** [review]
# Highlights ♻️ 

- [x] Well done, @MohamedHNoor  your movie count test is working.
- [x] I approved your code. Thank you for the hard work  ✅✅✅✅✅

**Pull request merged**